pages/profile/read/views/profile.jsx

```diff
@@ -35,7 +35,7 @@ class Profile extends React.Component {
     const profileRole = (establishments.find(est => est.id === estId) || {}).role;
     const pilIncomplete = pil && (pil.status === 'inactive' || pil.status === 'pending');
 
-    const hasPil = pilLicenceNumber;
+    const hasPil = pilLicenceNumber && pil;
```

Comment:
Is `pil` in this case a consolidated PIL record, and so there's no need here to take training PILs into account?

Reply:
yes